### PR TITLE
Todo list App features

### PR DESCRIPTION
**Pull request by Samkiroko**
- An amazing project to work on 
- on the app, you can create a project and add a to-do list
- The app was developed as  the guideline and the requirements
- live link[ click here](https://distracted-bell-c2ccd9.netlify.app/)

### CODE REVIEW

File: src/modules/project.js [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+import $ from 'jquery';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import 'jquery/dist/jquery.min';
+import 'jquery/dist/jquery.slim';
+import 'popper.js/dist/popper-utils.min';
+import 'bootstrap/dist/js/bootstrap.min';
+import '@fortawesome/fontawesome-free/js/fontawesome';
+import '@fortawesome/fontawesome-free/js/solid';
+import '@fortawesome/fontawesome-free/js/regular';
+import '@fortawesome/fontawesome-free/js/brands';
```
Comment (Oghenebrume50) [inline src/modules/project.js:22]: Looking at this, I can see that it can be destructured, :thinking:

File: src/modules/projectlist.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-plusplus */
+/* eslint-disable no-undef */
+/*  eslint-disable  no-unused-vars */
```
Comment (Oghenebrume50) [inline src/modules/projectlist.js:3]: you shouldn't be having unused variables if projectList and globalTaskId aren't used here, then this means you can optimize the code better and take them to the right location

File: src/modules/projectlist.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    } else {
+      ++i;
+    }
+  }
+  return arr;
+}
+
+export function getPtojectId(element) {
+  return target.parentNode.id;
+}
```
Comment (Oghenebrume50) [inline src/modules/projectlist.js:21]: [optional] define the functions normally, then great and export object with just the name of each function, cleaner code :slightly_smiling_face:

File: src/modules/task.js [hunk 4]
```diff
@@ -0,0 +1,345 @@
...
+import $ from 'jquery';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import 'jquery/dist/jquery.min';
+import 'jquery/dist/jquery.slim';
+import 'popper.js/dist/popper-utils.min';
+import 'bootstrap/dist/js/bootstrap.min';
+import '@fortawesome/fontawesome-free/js/fontawesome';
+import '@fortawesome/fontawesome-free/js/solid';
+import '@fortawesome/fontawesome-free/js/regular';
+import '@fortawesome/fontawesome-free/js/brands';
```
Comment (Oghenebrume50) [inline src/modules/task.js:21]: see previous comment about this :slightly_smiling_face:

**Comment by Samkiroko** [comment]
- remove the js library I had used
- refactored the whole code

**Pull request merged**